docs/releases/v2023.2.rst

```diff
@@ -0,0 +1,174 @@
+Gluon 2023.2
+============
+
+Important notes
+---------------
+
+Upgrades to v2023.2 and later releases are only supported from releases v2021.1 and later.
```

Comment:
```suggestion
Upgrades to v2023.2 and later releases are only supported from releases v2022.1 and later.
```

At least because of the EFI Bug upgrades will only be supported from v2022.1.